docs/guides/create_service_existing_agent.md

```diff
@@ -1 +1,212 @@
-This tutorial will be added soon.
\ No newline at end of file
+The typical use case that this guide intends to illustrate is where a service owner wants to use already developed agents to run its own service, possibly customizing it through the service parameters. For example, an oracle agent service with existing agents, where the service owner specifies
+
+  - what exchanges the agents should feed the information from, and
+  - what blockchain the service is reporting to.
+
+##What you will learn
+In this guide, we will show how to:
+
+  - Create a service definition using an already available agent on the  [IPFS](https://ipfs.io/).
+  - Test the service using a local deployment.
+  - Publish the service on the [IPFS](https://ipfs.io/).
+  - Register the service in the on-chain protocol. We will be using the [Görli testnet](https://goerli.net/).
+  - Deploy the registered service.
+
+If you recall the [overview of the development process](./overview_of_the_development_process.md), this roughly consists in steps 4, 5, and 6. For illustration purposes, we will also be using the agents from the [Hello World agent service](../hello_world_agent_service.md) and we will create a new (but functionally equivalent) "Hello World 2 agent service". To complete all the steps in this guide, you should have a [Görli testnet](https://goerli.net/) wallet address (e.g., [Metamask](https://metamask.io/)) with some GörliETH funds in it.
+
+##Step-by-step instructions
+
+Before starting this guide, ensure that your machine satisfies the [framework requirements](./quick_start.md#requirements) and that
+you have followed the [setup instructions](./quick_start.md#setup). As a result you should have a Pipenv workspace folder.
+
+1. **Identify the IPFS hash of the agent.** This can be some agent with the desired functionality for which you already know the hash, or you can browse it in the repository of a published agent. For this example, we consider the `hello_world` agent, whose hash is
+
+    ```
+    valory/hello_world:0.1.0:bafybeicealdcbxjdejskntddizntwqmlpyxa2ujaxnw2cgy73x3swldwcq
+    ```
+
+    If you want, you can browse the agent contents stored in the IPFS [here](https://gateway.autonolas.tech/ipfs/bafybeicealdcbxjdejskntddizntwqmlpyxa2ujaxnw2cgy73x3swldwcq/hello_world/).
+
+    !!!note
+        Future releases of the {{open_autonomy}} framework will provide convenient commands to browse and discover existing registered components. For now, we assume that we already know the IPFS hash of the agent.
+
+2. **Create the service definition.** Create an empty folder with the service name (e.g., `hello_world_2_service`). Inside that folder, create a `README.md` file, where you can write a description of the service, and a service definition file `service.yaml`, where the agent IPFS hash must be specified:
```

Comment:
Why not just `hello_world`?

Reply:
not to confuse the reader with the existing hello_woprld